part-up/part-up
# Optimize mytribes menu

A couple of optimizations have been made to the mytribes menu:
- Limit the amount of fields returned in the request;
- Only fetch data that's not present yet;

The following requirements still apply:
- Don't show archived networks / partups;
- Show all networks the user is a member of;
- Show networks the user is not a member of, but the user is involved in a partup of that network;
- Show all upper partups;
- Show all supporter partups;

The following test scenario's have been followed:
1. Login to the admin user;
    - ING closed tribe: Admin is member but not involved in partup;
2. Become supporter in the Lifely -> 'Organize Meteor' partup
    - open menu, it should now appear;
3. Request to become an upper in the organize meteor partup
4. Login to the default user on a different browser;
    - Accept the request (via notifications);
5. Open menu with default user and check the ING public tribe
    - user is not a member, click to go to the new tribe landing page to confirm.
6. Switch back to admin and re-open menu
    - The partup should be moved but no request should be made.
7. Switch roles to supporter again (cog wheel) and re-open menu
    - Check if partup is moved again, no request should be made.
8. Leave the partup alltogether
    - It should be removed from the menu, no request should be made.

Closes #1131
